src/routes/about/+page.js

```diff
@@ -0,0 +1,352 @@
+import { tooltip } from 'svooltip';
+
+const dataSrc = `https://api.github.com/repos/spyder-ide/spyder/contributors?per_page=100`;
+
+const currentContributors = [
+  {
+    id: 365293,
+    name: "Carlos Córdoba",
+    role: "Lead mantainer",
+    tooltip: [
+      {
+        list: [
+           "Maintainer since 2013"
+        ]
+      },
+    ]
+  },
+  {
+    id: 16781833,
+    name: "Daniel Althviz",
+    role: "Co-mantainer",
+    tooltip: [
+      {
+        list: [
+          "Release manager",
+          "Maintainer of QtPy and QtAwesome",
+          "Manager of NumFOCUS small development grants"
+        ]
+      }
+    ]
+  },
+  {
+    id: 17051931,
+    name: "C.A.M. Gerlach",
+    role: "Core developer",
+    avatar_url: "/assets/authors/camgerlach/pic.webp",
+    tooltip: [
+      {
+        list: [
+          "Docs maintainer",
+          "Technical writer"
+        ]
+      }
+    ]
+  },
+  {
+    id: 6740194,
+    name: "Quentin Peter",
+    role: "Core developer",
+    tooltip: [
+      {
+        list: [
+          "Re-architecture communications between Spyder and Spyder-kernels",
+          "Creator of the Debugger pane",
+          "Many improvements to the IPython console"
+        ]
+      }
+    ]
+  },
+  {
+    id: 9618975,
+    name: "Ryan Clary",
+    role: "Core developer",
+    tooltip: [
+      {
+        list: [
+          "Maintainer of the standalone installers"
+        ]
+      }
+    ]
+  },
+  {
+    id: 7941918,
+    name: "Jitse Niesen",
+    role: "Core developer",
+    tooltip: [
+      {
+        list: [
+          "External plugins maintainer"
+        ]
+      }
+    ]
+  },
+  {
+    id: 42411448,
+    name: "Juan Sebastian Bautista",
+    role: "Junior Developer",
+    tooltip: [
+      {
+        list: [
+          "Improvements to the Windows installer, QtConsole and Spyder's UI/UX"
+        ]
+      }
+    ]
+  },
+  {
+    id: 5204788,
+    name: "Hendrik D. Louzada",
+    role: "Junior Developer",
+    tooltip: [
+      {
+        list: [
+          "Backend for the Remote client plugin"
```

Comment:
```suggestion
          "Developed backend for the remote client plugin"
```

A bit clearer as to the fact that he developed it from scratch and consistent casing